lib/views/home_content/challenge/challenge_card.dart

```diff
@@ -0,0 +1,84 @@
+import "package:flutter/material.dart";
+import "package:flutter_svg/svg.dart";
+import "package:proxima/models/ui/challenge_card_data.dart";
+import "package:proxima/utils/ui/key_value_list_generator.dart";
+
+class ChallengeCard extends StatelessWidget {
+  static const challengeGroupIconKey = Key("challenge_group_icon");
+  static const challengeSingleIconKey = Key("challenge_single_icon");
+
+  static const _opacityWhenChallengedFinished = 0.3;
+  static const _challengeGroupAsset = "assets/icons/challenge_group.svg";
+  static const _challengeSingleAsset = "assets/icons/challenge_single.svg";
+
+  final ChallengeCardData challenge;
+
+  const ChallengeCard(this.challenge, {super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    KeyValueListGenerator listGenerator = KeyValueListGenerator(
+      style: DefaultTextStyle.of(context).style,
+    );
+
+    if (!challenge.isFinished) {
+      listGenerator.addPair("Distance to post", "${challenge.distance} meters");
+    } else {
+      listGenerator.addLine("Challenged finished!");
+    }
+    if (!challenge.isGroupChallenge) {
+      listGenerator.addPair("Time left", "${challenge.timeLeft} hours");
+    }
+    listGenerator.addPair("Reward", "${challenge.reward} Centauri");
+
+    final icon = Opacity(
+      opacity: 0.8, // to make the icon less prominent
```

Comment:
The PR is really clean so I try to be a little more picky here :) 

To keep consistency, I think that we can extract the `0.8` into a private static const as you did with `_opacityWhenChallengedFinished` 😄